order/src/main/java/com/caldevsupplychain/order/controller/OrderControllerImpl.java

```diff
@@ -0,0 +1,79 @@
+package com.caldevsupplychain.order.controller;
+
+import java.util.List;
+
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import org.apache.shiro.authz.annotation.RequiresPermissions;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.BindException;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.caldevsupplychain.account.annotation.RequiresJwtAuthentication;
+import com.caldevsupplychain.account.service.AccountService;
+import com.caldevsupplychain.common.exception.ApiErrorsExceptionHandler;
+import com.caldevsupplychain.common.type.ErrorCode;
+import com.caldevsupplychain.common.ws.ApiErrorsWS;
+import com.caldevsupplychain.common.ws.ErrorWS;
+import com.caldevsupplychain.order.service.OrderService;
+import com.caldevsupplychain.order.util.OrderMapper;
+import com.caldevsupplychain.order.validator.OrderValidator;
+import com.caldevsupplychain.order.vo.OrderWS;
+import io.swagger.annotations.Api;
+
+@Slf4j
+@RestController
+@AllArgsConstructor
+@Api(value="/api/v1/orders", description = "Order API")
+@RequestMapping("/api/v1/orders")
+public class OrderControllerImpl implements OrderController {
+
+	private OrderService orderService;
+	private AccountService accountService;
+
+	private OrderValidator orderValidator;
+
+	private OrderMapper orderMapper;
+
+	private ApiErrorsExceptionHandler apiErrorsExceptionHandler;
+
+	@PostMapping("/order")
```

Comment:
don't need `/order`. the endpoint `/api/v1/orders` for create is good.

Reply:
fixed